tests/trello/gui/trello_gui_board.robot

```diff
@@ -0,0 +1,61 @@
+*** Settings ***
+Documentation    Board Tests
+Resource         resources/LoginPage.resource
+Resource         resources/BoardsPage.resource
+Resource         resources/components/TopBar.resource
+Resource         resources/components/ShownMenu.resource
+
+*** Test Cases ***
+create a board
+    [Tags]    smoke    gui    board.create
+    Open Browser                     ${LOGIN_URL}    ${BROWSER}    remote_url=http://selenium-hub:4444
```

Comment:
The Keyword "Open Browser From Environment" is already in `master`, we should use that keyword, so we're able to run the test both locally and with docker.